apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -20,7 +20,7 @@
         "title": "Custom Command",
         "type": "object",
         "additionalProperties": false,
-        "required": [ "name", "summary", "documentation", "parallelized" ],
+        "required": [ "name", "summary", "documentation", "parallelized", "optional" ],
```

Comment:
>"optional" [](start = 74, length = 10)

making this required is technically a breaking change. I think we should let "optional" be optional and have it default to "false"